src/routes/sessions/[uuid]/.SDKHeader.svelte

```diff
@@ -0,0 +1,3 @@
+<svelte:head>
+  <script crossorigin src="https://unpkg.com/webex@^2/umd/webex.min.js"></script>
```

Comment:
For performance reasons, it would be better to bundle the SDK as part of the build instead of relying over a CDN.

Reply:
I will create the JIRA for this to experiment SDK bundling in current vite configuration. Last time I checked vite threw some errors bundling Webex SDK.

Reply:
I had some luck using [node-polyfills for vite](https://www.npmjs.com/package/vite-plugin-node-polyfills).